client/app/domain/email-obfuscation/email-obfuscation.html

```diff
@@ -0,0 +1,40 @@
+<div>
```

Comment:
```suggestion
```

Reply:
This is made in order to have a single root element. Probably not necessary with AngularJS

Reply:
This is needed for `Directive`s but not for `Component`s :)

Reply:
In some other frameworks (I'm thinking of Vue and React)  it is also required. Therefore I don't know if we could benefit from adopting this practice

Reply:
Interesting, but this rises issues, what if the template of my component is:
```html
<td>Foo</td>
<td>Bar</td>
```

What am I supposed to surround it with ? A `tr` can't contain a `div`.

I know it's far fetched but there must be technical limitations to this